src/display_list_items.js

```diff
@@ -0,0 +1,27 @@
+const listHtml = (result) => `
+    <img src="${result.url}" alt="image">
+            <button id="comment-${result.id}">Comments</button>
+    `;
+
+const appendListItem = (result) => {
+  const listItem = document.createElement('li');
+  listItem.innerHTML = listHtml(result);
+  listItem.className = 'centered-column';
+  console.log(listItem);
+  const displayItems = document.querySelector('#display-items');
+  displayItems.append(listItem);
+};
+
+const displayItems = async () => {
+  const { results } = await (await fetch('https://pokeapi.co/api/v2/pokemon?limit=6&offset=50')).json();
+  let count = 1;
+  results.forEach(async (result) => {
+    const data = await (await fetch(result.url)).json();
+    result.url = data.sprites.other.dream_world.front_default;
+    result.id = count;
+    count += 1;
+    appendListItem(result);
+  });
+};
+
+displayItems();
```

Comment:
Here you mix DOM manipulation, Object, and API calls.
I think is better if this file should be divided into 2 or 3 files and each file have his specific goal.